todo_app/data/Trello.py

```diff
@@ -0,0 +1,119 @@
+from todo_app.data.ResponseFilter import ResponseFilter
+from todo_app.ListUtils import swap_key_in_list, combine_lists
+import requests
+import os
+
+
+class trello:
+    api_key = os.environ.get('API_KEY')
+    api_token = os.environ.get('API_TOKEN')
+    trello_board_id = os.environ.get('BOARD_ID')
+
+    todo_name = 'To Do'
+    in_progress = 'Doing'
+    done = 'Done'
+    todo_id = None
+    in_progress_id = None
+    done_id = None
+
+    base_url = 'https://api.trello.com/1/'
+
+    headers = {'key': api_key, 'token': api_token}
+
+    fields_from_cards = {'fields': f'{ResponseFilter.name},{ResponseFilter.id_list}'}
+
+    get_cards_header = headers | fields_from_cards
+
+    get_all_cards_url = f'{base_url}boards/{trello_board_id}/cards'
+
+    fields_from_list = {ResponseFilter.fields: f'{ResponseFilter.name},idBoard'}
+
+    get_lists_headers = headers | fields_from_list
+
+    get_lists_url = f'{base_url}boards/{trello_board_id}/lists'
+    cards_url = f'{base_url}cards'
+
+    lists_from_board = None
+
+    @classmethod
+    def get_lists(cls):
+        if cls.lists_from_board is None:
+            lists = requests.get(
+                cls.get_lists_url, params=cls.get_cards_header).json()
+            lists = swap_key_in_list(
+                lists, ResponseFilter.id, ResponseFilter.id_list)
+            cls.lists_from_board = swap_key_in_list(
+                lists, ResponseFilter.name, ResponseFilter.status)
+
+        return cls.lists_from_board
+
+    @classmethod
+    def get_all_list_ids(cls, list_name):
+        lists = cls.get_lists()
+        return next((item for item in lists if item[ResponseFilter.status] == list_name), None)[ResponseFilter.id_list]
```

Comment:
Since you created `search_list`, you could use it here:
```suggestion
        return search_list(cls.get_lists(), ResponseFilter.status, list_name)[ResponseFilter.id_list]
```